packages/ActionBar/src/components/filter.js

```diff
@@ -0,0 +1,13 @@
+export const Filter = () => {
+  console.error(
+    "The <Filter /> component in @paprika/action-bar has been deprecated. Please use the package @paprika/filter directly."
```

Comment:
do we have anyone using this in production? if we do maybe would be better to do `throw new Error("...")` so they are forced to see the error?

Reply:
No, nothing in production, but good point.